Client/Telemetry/CorePing.swift

```diff
@@ -94,6 +94,13 @@ class CorePing: TelemetryPing {
             out["defaultMailClient"] = chosenEmailClient as AnyObject?
         }
 
+        if #available(iOS 11.0, *) {
+            let strength = prefs.stringForKey(ContentBlockerHelper.PrefKeyStrength) ?? "unchanged"
```

Comment:
I dont think we need to do this. To keep things consistent with the rest of the core ping. Only send the option if it has changed.

Reply:
Agreed, I updated the PR